pkg/vm/engine/types.go

```diff
@@ -158,6 +161,105 @@ func (*PropertiesDef) tableDef() {}
 func (*ClusterByDef) tableDef()  {}
 func (*ConstraintDef) tableDef() {}
 
+func (def *CommentDef) ToPBVersion() TableDefPB {
```

Comment:
I am not sure why do we need all these to pb stuff.     all we need is encode/decode from []bytes and these code should be from protogen, no?

Reply:
`engine.TableDef` are referenced by many other structure, such as [DelTableDefReq](https://github.com/matrixorigin/matrixone/blob/352fa7db1643661ea83671af2088fda57a26418d/pkg/vm/engine/memoryengine/operations.go#L220), I just want to make use of PB in order to limit manual formatting, so `ToPBVersion` was introduced. In our project, there's still other cases that translate interface based structure to PB generated structure, such as [toPBBatch](https://github.com/matrixorigin/matrixone/blob/352fa7db1643661ea83671af2088fda57a26418d/pkg/vm/engine/disttae/tools.go#L747).